File: articles/algorithmic-fairness-and-computational-justice/cpp/fairness_gap_score.cpp
#include <algorithm>
#include <iostream>
#include <vector>

int main() {
    std::vector<double> rates = {0.42, 0.31, 0.36};
    double min_rate = *std::min_element(rates.begin(), rates.end());
    double max_rate = *std::max_element(rates.begin(), rates.end());
    std::cout << "selection_gap=" << max_rate - min_rate << "\n";
    return 0;
}

File: articles/algorithmic-fairness-and-computational-justice/go/fairness_gap_score.go
package main

import "fmt"

func main() {
	rates := []float64{0.42, 0.31, 0.36}
	minRate := rates[0]
	maxRate := rates[0]
	for _, value := range rates {
		if value < minRate {
			minRate = value
		}
		if value > maxRate {
			maxRate = value
		}
	}
	fmt.Printf("selection_gap=%.4f\n", maxRate-minRate)
}
